SPECS/python-paramiko/python-paramiko.spec

```diff
@@ -0,0 +1,736 @@
+%global srcname paramiko
+
+Name:          python-%{srcname}
+Version:       2.11.0
+Release:       2%{?dist}
+Summary:       SSH2 protocol library for python
+License:       LGPLv2+
+Distribution:  Mariner
+Vendor:        Microsoft Corporation
+URL:           https://github.com/paramiko/paramiko
+Source0:       %{url}/archive/refs/tags/%{version}.tar.gz#/%{srcname}-%{version}.tar.gz
+
+# Skip tests that would fail without SHA-1 signing support in backend (e.g. on EL-9)
+# Can be removed when https://github.com/paramiko/paramiko/pull/2057/ is released
+Patch1:        0001-Mark-new-tests-that-require-SHA1.patch
+
+# Skip tests requiring invoke if it's not installed
+# Can be removed when https://github.com/paramiko/paramiko/pull/1667/ is released
+Patch2:        0002-Skip-tests-requiring-invoke.patch
+
+# Remove pytest-relaxed, which depends on pytest4
+# Can be removed when https://github.com/paramiko/paramiko/pull/1665/ is released
+Patch3:        0003-remove-pytest-relaxed-dep.patch
+
+# Avoid use of deprecated python-mock by using unittest.mock instead
+# Can be removed when https://github.com/paramiko/paramiko/pull/1666/ is released
+Patch4:        0004-remove-mock-dep.patch
+
+BuildArch:     noarch
+
+%global paramiko_desc \
+Paramiko (a combination of the Esperanto words for "paranoid" and "friend") is\
+a module for python 2.3 or greater that implements the SSH2 protocol for secure\
+(encrypted and authenticated) connections to remote machines. Unlike SSL (aka\
+TLS), the SSH2 protocol does not require hierarchical certificates signed by a\
+powerful central authority. You may know SSH2 as the protocol that replaced\
+telnet and rsh for secure access to remote shells, but the protocol also\
+includes the ability to open arbitrary channels to remote services across an\
+encrypted tunnel (this is how sftp works, for example).
+
+%description
+%{paramiko_desc}
+
+%package -n python3-%{srcname}
+Summary:       SSH2 protocol library for python
+BuildRequires: python3-sphinx
+BuildRequires: python3-devel
+BuildRequires: python3-bcrypt
+BuildRequires: python3-cryptography
+BuildRequires: python3-pyasn1
+BuildRequires: python3-pynacl
+BuildRequires: python3-setuptools
+BuildRequires: python3-six
+%if %{with_check}
+BuildRequires:  python3-atomicwrites
+BuildRequires:  python3-attrs
+BuildRequires:  python3-docutils
+BuildRequires:  python3-pluggy
+BuildRequires:  python3-pygments
+BuildRequires:  python3-six
+BuildRequires:  python3-pip
+BuildRequires:  python3-pytest
+%endif
```

Comment:
We could just use pip to install all of these- that's fine for check builds, which have internet enabled.

Reply:
Done